### PR TITLE
Added update available message.

### PR DESCRIPTION
**Pull request by KozmikNano**
I have added a new file and dependency. rss-parser and check.js work together to detect the latest release version then compares it to the current version. Before you release a new version you have to change the line number 5 to match the next release number. I have already got it setup for the next commit witch should include this.

### CODE REVIEW

File: check.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+let Parser = require('rss-parser');
+let parser = new Parser();
+const { exec} = require('child_process');
+const boxen = require('boxen');
+var cv = "v0.1.17";
```
Comment (KozmikNano) [inline check.js:5]: If you are confused just tell me and i can walk you through it.Not editing this one line could cause an issue with starting even when at latest version!!!!!

Comment (KozmikNano) [reply check.js:5]: Line 5 is the one i am talking about.

**Comment by themoonisacheese** [comment]
Good work, although having to update the version number manually is tedious and easy to miss. The current version number is always up to date and available in package.json

**Comment by KozmikNano** [comment]
Ok I will look into fixing this.

**Pull request closed without merging**